randovania/games/dread/pickup_database/pickup-database.json

```diff
@@ -884,18 +884,19 @@
         "Speed Booster Upgrade": {
             "gui_category": "misc",
             "hint_features": [
-                "major",
+                "expansion",
                 "misc"
             ],
             "model_name": "item_speedboostupgrade",
             "offworld_models": {
                 "am2r": "sItemSpeedBoosterUpgradeDread"
             },
             "preferred_location_category": "minor",
-            "description": "Speed Booster Upgrades reduce the amount of time it takes Speed Booster to activate by 0.25 seconds (for each upgrade collected).\nThe minimum charge time is 0.25s.",
+            "description": "Each Speed Booster Upgrade reduces the amount of time Speed Booster takes to charge by one quarter of a second.\nDue to game limitations, the minimum allowed charge time is approximately half a second.",
```

Comment:
While im here: i personally find reading 0.25 and 0.5 easier than writing those out into words. I would assume its also easier for other non-english natives.
But idk if its significant enough to actually get changed.

Reply:
I didn't want to use exact values because I want to avoid the illusion of it being too precise. Given that the final upgrade reduces by _less than_ 0.25 seconds, and the minimum time is _slightly more than_ 0.5 seconds, I figured that approximated language would be better.

Reply:
1) could just use "approximately 0.25" then
2) are these significant enough where someone would actually notice it?

Reply:
I'm not sure if has anything with not being a native english speaker, but I also think writing out the numbers with words is a bit jarring. Especially considering they were written with numbers before. Also, I don't think using words is a meaningful way to dampen the notion of the values being exact.

Reply:
Is it even necessary to mention how much it changes it by? Why not just say that the charge time is reduced?